src/tests/test_package_loading.py

```diff
@@ -569,8 +569,7 @@ def write(self, base_dir: Path) -> None:
                 (dist_info_dir / key).write_text(value)
         with (dist_info_dir / "METADATA").open("w") as f:
             f.write(
-                f"Metadata-Version: 2.1\nName: {self.name}\n"
-                f"Version: {self.version}\n"
+                f"Metadata-Version: 2.1\nName: {self.name}\nVersion: {self.version}\n"
```

Comment:
Still waiting on a fix for this. I'd like the three lines of the string to appear on separate source lines.